_targets.R

```diff
@@ -19,15 +23,13 @@ list(
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
   # TODO: PULL SITE DATA HERE
-  tar_target(wi_data, get_site_data(sites_info = oldest_active_sites,
-                                    state = 'WI',
-                                    parameter = parameter)),
-  tar_target(mn_data, get_site_data(sites_info = oldest_active_sites,
-                                    state = 'MN',
-                                    parameter = parameter)),
-  tar_target(mi_data, get_site_data(sites_info = oldest_active_sites,
-                                    state = 'MI',
-                                    parameter = parameter)),
+  tar_map(
+    ## task names passed into tar_map() via arg. 'values='
+    values = tibble(state_abb = states),
+    tar_target(nwis_data, get_site_data(oldest_active_sites, state_abb, parameter))
+    # Insert step for tallying data here
+    # Insert step for plotting data here
+  ),
 
   # Map oldest sites
   tar_target(
```

Comment:
Note that when I built the pipeline with the new state added to the (`IL`), I got the following output. 

<img width="423" alt="image" src="https://user-images.githubusercontent.com/36547359/145876208-12eee9dc-aed4-49d6-b65d-3971d5903129.png">

Why is `nwis_data_IL` built after site_map_png, even though `oldest_active_sites` includes the new state `IL`.

Reply:
I am not sure why. However, there is no reason not to do it in principle, since `site_map_png` is dependent only on `oldest_active_sites`, as is also the case for all of the `nwis_data_XX` targets. Since they are independent of each other, then can be run in any order! In practice, I have noticed that targets sometimes seems to change up the order or introduce a little bit of randomness when a set of targets all have the same prerequisites.

![image](https://user-images.githubusercontent.com/13890601/145907972-00f33fe0-2dbe-4643-8207-ba058a5115fa.png)